community/examples/htc-htcondor.yaml

```diff
@@ -104,45 +104,32 @@ deployment_groups:
     outputs:
     - internal_ip
 
-  - id: htcondor_startup_execute_point
-    source: modules/scripts/startup-script
-    settings:
-      runners:
-      - $(htcondor_secrets.execute_point_runner)
-      - $(htcondor_base.execute_point_runner)
-
   # the HTCondor modules support up to 2 execute points per blueprint
   # if using 1, it may use Spot or On-demand pricing
   # if using 2, one must use Spot and the other must use On-demand (default)
   - id: htcondor_execute_point
     source: community/modules/compute/htcondor-execute-point
     use:
     - network1
-    - htcondor_startup_execute_point
+    - htcondor_secrets # must be ordered before htcondor_base
+    - htcondor_base
     settings:
       instance_image:
         project: $(vars.project_id)
         family: $(vars.new_image_family)
       min_idle: 2
-      service_account:
-        email: $(htcondor_base.execute_point_service_account_email)
-        scopes:
-        - cloud-platform
 
   - id: htcondor_execute_point_spot
     source: community/modules/compute/htcondor-execute-point
     use:
     - network1
-    - htcondor_startup_execute_point
+    - htcondor_secrets # must be ordered before htcondor_base
```

Comment:
I'm having a hard time to understand why order matters here. I guess it's to shadow or oder (for list variables) outputs.
The only overlap in output names of `htcondor_secrets` &   `htcondor_base` is `central_manager_runner`, but 
`htcondor-execute-point` doesn't take `central_manager_runner` as an input.

Based on [4804b21](https://github.com/GoogleCloudPlatform/hpc-toolkit/pull/1574/commits/4804b21f50d2bd3021e3fb1211d96891f3f0b58e)

Reply:
Good eye. This PR actually resolves the ordering problem that was present during earlier stage of develop.

The underlying issue is that the secrets must be staged prior to starting the HTCondor daemons. I will address this.

Reply:
See 77dd8ae1